src/libraries/System.Private.CoreLib/src/System/String.Manipulation.cs

```diff
@@ -2360,28 +2360,6 @@ public unsafe string Trim(params char[]? trimChars)
             }
         }
 
-        /// <summary>
-        /// Removes all leading and trailing occurrences of a set of characters specified in a span from the current string.
-        /// </summary>
-        /// <param name="trimChars">A span of Unicode characters to remove.</param>
-        /// <returns>
-        /// The string that remains after all occurrences of the characters in the <paramref name="trimChars"/> parameter are removed from the start and end of the current string.
-        /// If <paramref name="trimChars"/> is empty, white-space characters are removed instead.
-        /// If no characters can be trimmed from the current instance, the method returns the current instance unchanged.
-        /// </returns>
-        public unsafe string Trim(params ReadOnlySpan<char> trimChars)
```

Comment:
Is there anywhere we were using this that will now revert to allocating a char[]?

Reply:
Aside from the one break in WPF, the only place I've seen from grep.app is

https://github.com/dotnet/wpf/blob/673c35418e5416e486e6651b69b89712a36fcf5c/src/Microsoft.DotNet.Wpf/src/System.Xaml/System/Xaml/Parser/XamlText.cs#L247-L257

but source.dot.net says that's already the `char[]` one